### DISCUSSION TITLE
tana user struggling to get openai scripts working (win 10)

### DISCUSSION THREAD

**Discussion by bitbutter** [johnlindquist/kit#991] [Errors] [answered]
hi, i'm on windows 10, trying to get an example openAI script to work as expected.

i have an openAI account. I have successfully generated a key, and entered it at the script prompt when run.

when i run the OpenAI Playground script and enter a prompt i get the following error, and i'm not sure how to proceed:
![image](https://user-images.githubusercontent.com/52042/212909509-528ce135-dddf-4694-a95e-560d46376b75.png)

any help or guidance much appreciated!

**Comment by bitbutter** [comment, marked as answer]
i had upgraded to a pay-as-you-go openAI account just prior to trying this. i guess it took a while before the system knew about my credits, it's working as expected now.

**Reply by johnlindquist** [reply]
Awesome, thanks for reporting back. Happy to hear it's working.
